pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -106,9 +106,10 @@ func (c *ConfigMapUnpacker) job(cmRef *corev1.ObjectReference, bundlePath string
 							},
 						},
 						{
-							Name:    "pull",
-							Image:   bundlePath,
-							Command: []string{"/util/cpb", "/bundle"}, // Copy bundle content to its mount
+							Name:            "pull",
+							Image:           bundlePath,
+							ImagePullPolicy: "Always",
```

Comment:
Should we be worried about the consequences of pulling an image in a bundle everytime an operator is deployed? If operators are installed in multiple namespaces this is expensive.

Reply:
Yes, this is essentially the tradeoff

Reply:
I don't think this is such a big deal given the bundle image is just scratch image so it is small. Plus, unless there are multiple new installations of the same operator in multiple namespaces (that is different with MultiNamespace mode), then this scenario doesn't happen. The common pattern of operator installation so far has been pretty much singleton.